src/lib/SX12xxDriverCommon/SX12xxDriverCommon.h

```diff
@@ -44,8 +44,18 @@ class SX12xxDriverCommon
 
     /////////////Packet Stats//////////
     int8_t LastPacketRSSI;
+    int8_t LastPacketRSSI2;
     int8_t LastPacketSNRRaw; // in RADIO_SNR_SCALE units
 
+    bool isFirstRxIrq = true;
+
+#if defined(DEBUG_RCVR_DUAL_RSSI)
+    uint16_t irq_count[4];
+    uint16_t fail_count;
+    uint16_t telem_count[2];
+    int16_t snr_sum[2];
```

Comment:
It might be nicer to create a struct for this rather than arrays!

Reply:
Wilco

Reply:
Done